vividus-tests/src/main/resources/story/integration/CodeSteps.story

```diff
@@ -10,11 +10,19 @@ Given I am on a page with the URL '${vividus-test-site-url}'
 When I execute javascript `return JSON.stringify(window.performance.timing)` and save result to scenario variable `timings`
 Then number of JSON elements from `${timings}` by JSON path `$.connectStart` is = 1
 
+
 Scenario: Step verification: "When I perform async javascript '$jsCode' and save result to the '$scopes' variable '$variableName'"
 !-- Deprecated
 When I perform async javascript 'arguments[0](JSON.stringify(window.performance.timing))' and save result to the 'scenario' variable 'timings'
 Then number of JSON elements from `${timings}` by JSON path `$.connectStart` is = 1
 
+
 Scenario: Step verification: "When I execute async javascript `$jsCode` and save result to $scopes variable `$variableName`"
 When I execute async javascript `arguments[0](JSON.stringify(window.performance.timing))` and save result to scenario variable `timings`
 Then number of JSON elements from `${timings}` by JSON path `$.connectStart` is = 1
+
+
+Scenario: Step verification: "When I execute javascript `$jsCode` with arguments:$args"
+Then number of elements found by `xpath(//img)` is = `1`
+When I execute javascript `document.querySelector('[name="vividus-logo"]').remove()` with arguments:
+Then number of elements found by `xpath(//img)` is = `0`
```

Comment:
should we refresh the page at the end to return it to its initial state? the change is extremely low but we might accidentally add one more scenario after this one that will rely on the logo element and it will fail because we didn't notice logo remove operation in previous scenario

Reply:
If we add an extra scenario then there should be refresh, now its redundant